.babelrc.js

```diff
@@ -0,0 +1,46 @@
+module.exports = {
```

Comment:
should not it be babel.config.js ?

Reply:
Got this from here:
https://babeljs.io/docs/en/config-files

Seems to work..